tests/Fixture/ToastMessage.elm

```diff
@@ -0,0 +1,13 @@
+module Fixture.ToastMessage exposing (toastFuzzer)
+
+import Fuzz exposing (Fuzzer, int, list, string)
+import Json.Encode as Encode
+import Message.Toast exposing (Toast)
+
+
+toastFuzzer : Fuzzer Toast
+toastFuzzer =
+    Fuzz.map3 Toast
+        (Fuzz.maybe Fuzz.string)
+        Fuzz.string
+        (Fuzz.maybe (Fuzz.constant Encode.null))
```

Comment:
Elm's testing library has property-based testing built in. This allows for the random generation of data to use in tests via "fuzzers". This describes a fuzzer that will create a `Toast` with random data in it. Tests using the fuzzer will test many random versions (100 by default) and if any of them fail, will try to reduce the data to a minimal failing test case. It's really nice for validating things like Encoders and Decoders where there are a lot of potential variations. There aren't a lot of possibilities here, but it may grow in the future, and it's easy enough to reduce the number of fuzz iterations used in a test.